plutus-core/executables/plutus/AnyProgram/Run.hs

```diff
@@ -19,7 +19,6 @@ import Types
 import UntypedPlutusCore as UPLC
 import UntypedPlutusCore.Evaluation.Machine.Cek as UPLC
 
-import Data.Foldable
```

Comment:
I'm going to occasionally update `PlutusPrelude` so that we have less imports clutter.